src/jobs/jobs.service.ts

```diff
@@ -112,6 +113,7 @@ export class JobsService {
           ),
           statusUpdateJobDto.statusCode,
           statusUpdateJobDto.statusMessage!,
+          statusUpdateJobDto.jobResultObject!,
```

Comment:
In the StatusUpdateJobDto, both statusMessage and jobResultObject is decorated as optional, but here the both are set as must fields with exclmination mark.

Reply:
Thanks! `statusMessage` should be required and `jobResultObject` should be optional. Fixed it the latest commit (https://github.com/SciCatProject/scicat-backend-next/pull/1410/commits/c2af58544f3e8b30fae3664aff7f72a6eed35cf9)